SU-SWS/stanford-jumpstart-deployer
# DEVOPS 121: Update to entity 7.x-1.9 and drupal core 7.57 (5x branch)

# READY FOR REVIEW

# Summary
- Applying the following security releases for entity and drupal core:
https://www.drupal.org/SA-CORE-2018-001
https://www.drupal.org/sa-contrib-2018-013

# Needed By (Today or Mon.)

# Urgency
- 9

# Steps to Test

1. Pull this branch
2. Run `drush make production/product/jumpstart/jumpstart.make ~/Sites/devops121-jumpstart --working-copy --concurrency=4 -y` to be sure site builds.
3. Travis passed, see below.

# Affected Projects or Products
- Affects all sites on sites infrastructure

# Associated Issues and/or People
- https://stanfordits.atlassian.net/browse/DEVOPS-121
- https://github.com/SU-SWS/Stanford-Drupal-Profile/pull/98

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
